cmds/console/uart_linux.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2012-2017 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// This file contains support functions for UART io for Linux.
+package main
+
+import (
+	"os"
+)
+
+const (
+	data   = 0x3f8
+	status = 0x3fd
+	inRdy  = 1
+	outRdy = 0x40
+)
+
+type uart struct{}
```

Comment:
actually, the struct should have the offset into /dev/port, what was I thinking. Will change.